src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -179,6 +184,38 @@ public Executive(RuntimeCore runtimeCore, bool isFep = false)
         }
 
         /// <summary>
+        /// Bounce to an existing executive
+        /// </summary>
+        /// <param name="exeblock"></param>
+        /// <param name="entry"></param>
+        /// <param name="context"></param>
+        /// <param name="stackFrame"></param>
+        /// <param name="locals"></param>
+        /// <param name="fepRun"></param>
+        /// <param name="exec"></param>
+        /// <param name="breakpoints"></param>
+        /// <returns></returns>
+        public StackValue BounceUsingExecutive(
+           DSASM.Executive executive,
+           int exeblock,
+           int entry,
+           ProtoCore.Runtime.Context context,
+           StackFrame stackFrame,
+           int locals = 0,
+           bool fepRun = false,
+           DSASM.Executive exec = null,
+           List<Instruction> breakpoints = null)
+        {
+            if (stackFrame != null)
+            {
+                SetupAndPushBounceStackFrame(exeblock, entry, context, stackFrame, locals);
+                runtimeCore.DebugProps.SetUpBounce(exec, stackFrame.FunctionCallerBlock, stackFrame.ReturnPC);
+            }
+            executive.Execute(exeblock, entry, breakpoints);
+            return executive.RX;
+        }
```

Comment:
I will, but i will likeley get rid of Bounce() and just use BounceUsingExecutive.
This is hard to do now, because in the future, we may decide to deploy/execute a language block within a different VM... that will require a Bounce() whereas running local, we use only BounceUsingExecutive